Mod1/Lessons/Week4/IntroToIDE.md

```diff
@@ -32,9 +32,11 @@ We will follow these steps to download Visual Studio Community (a free IDE that
 ![](/Mod1/Images/Week4/VisualStudioDownloadButton.png)
```

Comment:
I don't think I see anywhere that you use the RemoveCodeLense image. I could see either adding a section for how to remove codelense later if you forgot to check it during set-up or removing the image. Either way.